src/v2_services/chat_service.py

```
@@ -153,10 +154,16 @@ async def translate_query(query: str) -> str:
         이 파이프라인은 | (파이프) 연산자로 각 단계를 연결합니다.
         데이터가 왼쪽에서 오른쪽으로 순차적으로 흐릅니다.
     """
-    chain = translation_prompt | llm | StrOutputParser()
-    translated = await chain.ainvoke({"query": query})
-    print(f"🔄 번역: '{query}' → '{translated}'")
-    return translated
+    try:
+        chain = translation_prompt | llm | StrOutputParser()
+        translated = await chain.ainvoke({"query": query})
+        if not translated or not translated.strip():
+            raise ValueError("번역 결과가 없습니다.")
```

Ruff (0.15.2)

[warning] Abstract `raise` to an inner function

(TRY301)